README.adoc

```diff
@@ -44,6 +44,10 @@ This project provides `SmallRyeClientTracingFeature` with tracing integration. T
 has to be registered to `ClientBuilder` in vendor specific implementation of `ClientTracingRegistrarProvider`.
 Client side tracing usually requires more components, for example OpenTracing-aware `AsyncExecutor`.
 
+=== MicroProfile Rest Client
```

Comment:
Any reason why SmallRye OpenTracing shouldn't also include the service file if it's providing the listener?

Reply:
Would the file be automatically packaged or does vendor have to put it into deployment manually? E.g. thorntail or wildfly? I think the would not be discovered automatically and vendors have to do add it manually anyways.

Reply:
There might be some vendors that need to add it to a deployment, but we also want this to be usable separately as well.

Reply:
This library would not work without additional configuration. There are different pieces which are required by vendors to implement and configure.

Reply:
Ok, but for this the library already provides an implementation, so why not add the service file too so that it doesn't need to be added if a vendor doesn't need to?

Reply:
I have added that file

Reply:
Thanks!